File: c/Lista3/Desafio1.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

main(){

	float Area, Base, Altura;

printf("Digite a base do retângulo: ");
scanf("%f", &Base);

printf("Digite a altura do retângulo ");
scanf("%f", &Altura);

Area = Base * Altura;
printf("A area do retângulo é:  %3.0f",Area);

}

File: js/lista3/Desafio1.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
}); 

/*
Desafio1- Escrever um programa para ler as dimensões de um retângulo (base e altura) pelo teclado:

Calcular e escrever a área do retângulo;

Fórmula: Area = base * altura.
*/

rl.question('Digite o valor da base: ', (base) => {
    rl.question('Digite o valor da altura: ', (altura) => {
      let area = base * altura;
     
        console.log(`O valor da área do retângulo é: ${area}`);
      rl.close();
    });
  });